src/locales/de.yaml

```diff
@@ -7,40 +7,40 @@ app:
       home: Dashboard
       add_printer: Drucker hinzufügen
       endstops: Endschalter
-      fans_outputs: Fans & Ausgänge
+      fans_outputs: Lüfter & Ausgänge
       jobs: Aufträge
-      macros: Macros
+      macros: Makros
       limits: Drucker Limits
       logs: Drucker Logs
       system_control: Systemsteuerung
-      runout_sensors: Runout Sensors
+      runout_sensors: Runout Sensoren
       status: Status
       temperature: Temperaturen
-      tool: Tool
+      tool: Druckkopf
       tune: Tune
       configure: Konfiguration
       interface: Interface
-      stats: Drucker Status
+      stats: Druckerstatus
       history: Auftragsverlauf
-      config_files: Konfigurations Dateien
+      config_files: Konfigurationsdateien
     label:
-      accel_to_decel: Accel zu Decel
-      acceleration: Acceleration
-      sqv: Square Corner Velocity
-      velocity: Velocity
-      api_url: API URL
+      accel_to_decel: Beschleunigen zu Bremsen
```

Comment:
Literal translation doesn't work too well here, maybe we can talk to Kevin what this parameter represents to find a better translation

Reply:
I am still not sure how to translate this properly. Any help would be appreciated.